pool/src/types.rs

```diff
@@ -44,37 +44,19 @@ const DANDELION_STEM_PROBABILITY: usize = 90;
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct DandelionConfig {
 	/// Choose new Dandelion relay peer every n secs.
-	#[serde = "default_dandelion_relay_secs"]
-	pub relay_secs: Option<u64>,
+	#[serde(default = "default_dandelion_relay_secs")]
```

Comment:
This is going to cause all kinds of conflicts with 1.1.0 branch.
It might be worth not touching the config here and just doing it on the 1.1.0 branch.
We fixed the "default" serde config like you do here, but we did not go the additional step and get rid of the options (which I agree is :+1:).

Reply:
@antiochp ok, I've reverted this commit

Reply:
I suspect `#[serde = "foo"]` vs. `#[serde(default = "foo")]` has bitten us and continues to bite us all over our config parsing code.